pyHS100/smartbulb.py

```diff
@@ -1,6 +1,9 @@
 from pyHS100 import SmartDevice
+import re
 from typing import Any, Dict, Optional, Tuple
 
+TPLINK_KELVIN = {'LB130': (2700, 9000),
+                 'LB120': (2700, 6500)}
```

Comment:
While you are at it, please add information also from other bulbs: https://www.tp-link.com/us/home-networking/smart-home/smart-bulbs

Reply:
done !

Reply:
How about LB110, LB110, LB200, and LB230 (although LBx series is new to me, no idea if it's supported, but it would make sense to have these already in place anyway)?